apps/ftu-ui-small-wc/src/assets/TEMP/temp-svgs/2d-ftu-kidney-nephron.svg

```diff
@@ -396,7 +396,7 @@
       }
 
       .cls-97 {
-        font-family: InterV-Italic, 'InterV Italic';
+        font-family: Inter;
         font-style: italic;
         font-variation-settings: ';
```

Comment:
What is this tick mark doing? :-O

Reply:
It was there before, I didn’t want to touch it 🤷‍♂️